tests/typescript.js

```diff
@@ -31,39 +31,22 @@ describe("typescript", () => {
 
             const scopeManager = analyze(ast);
 
-            expect(scopeManager.scopes).to.have.length(4);
+            expect(scopeManager.scopes).to.have.length(2);
```

Comment:
Does this seem correct? I _think_ this makes sense, since the first two are type definitions and so shouldn't actually create a new scope? Would love it if @bradzacher or @JamesHenry would be so kind to chime in!

Reply:
Yes. I think that https://github.com/eslint/typescript-eslint-parser/pull/540 has affected it.